IntegrationTests/SizeFlexibilityUpdateTest.js

```diff
@@ -25,16 +25,21 @@ let finalState = false;
 
 const SizeFlexibilityUpdateTest = createReactClass({
   displayName: 'SizeFlexibilityUpdateTest',
-  mixins: [Subscribable.Mixin],
+  _subscription: (null: ?EmitterSubscription),
 
   UNSAFE_componentWillMount: function() {
-    this.addListenerOn(
-      RCTNativeAppEventEmitter,
+    this._subscription = RCTNativeAppEventEmitter.addListenerOn(
```

Comment:
This should have actually been:
```typescript
this._subscription = RCTNativeAppEventEmitter.addListener(
```

There's no method called `RCTNativeAppEventEmitter.addListenerOn`. But it's no big deal, I patched it internally. 🙂

Reply:
I wasn't able to test it 😅. Thanks